DoSomethingArchive/legacy-website
# Adds global admin permissions

#### What's this PR do?
- Adds new permissions to the global admin role (somehow they were previous erased)
- Gives global admin access to tabs
#### How should this be manually tested?

Make a global admin account, verify they have access to everything outlined in #4949 
#### What are the relevant tickets?
#4949 (Not gonna say it fixes yet, wanna get this deployed somewhere Micah can run through being a global admin and make sure everything she needs right now is available)
